modules/ROOT/pages/veevavault/veevavault-connector-design-center.adoc

```diff
@@ -0,0 +1,60 @@
+= VeevaVault Connector Design Center Configuration - Mule 4
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+Anypoint Design Center's Flow Designer enables you to create apps visually. For more information, see the xref:design-center::fd-tour.adoc[Flow Designer Tour].
+
+To create an app in Flow Designer:
+
+* Configure the input source (trigger) for your app.
+* Add the connector as a component to process the input for the app.
+
+For information on Flow Designer field values, see the xref:./veevavault-connector-reference.adoc[VeevaVault Connector Reference].
+
+
+== Configure the Input Source Trigger
```

Comment:
I asked Diane about this template item just recently.
See Jan. 16th Slack convo.
We first refer to "input source (trigger)," as if they are two synonymous terms.
But this title refers to one thing: an input source trigger.
Which is correct?